test/WebJobs.Script.Tests/EndToEndTimeoutTests.cs

```diff
@@ -0,0 +1,229 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.ExceptionServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Host.Timers;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests
+{
+    public class EndToEndTimeoutTests
+    {
+        [Fact]
+        public async Task TimeoutTest_SyncFunction_Node()
```

Comment:
If your goal is to have the same timeout test run across all languages, just put the single test in the base class (like QueueTriggerToBlob, etc.) so it gets inherited and run?